src/models/bonw16/repairs.md

```diff
@@ -1,3 +1,283 @@
+
+- [Removing the bottom cover](#removing-the-bottom-cover)
+- [Replacing the battery](#replacing-the-battery)
+- [Replacing the RAM](#replacing-the-ram)
+- [Replacing an M.2/NVMe SSD](#replacing-an-m2nvme-ssd)
+- [Replacing the WiFi/Bluetooth module](#replacing-the-wireless-card)
+- [Replacing the CMOS battery](#replacing-the-cmos-battery)
+- [Replacing the speakers](#replacing-the-speakers)
+- [Replacing the fans/heatsink/thermal paste](#replacing-the-cooling-system)
+- [Replacing the keyboard](#replacing-the-keyboard)
+
+## Removing the bottom cover:
+
+Removing the cover is required to access the internal components. Prior to removing the cover, ensure the AC power is unplugged and all peripherals (such as USB drives or Kensington locks) are unplugged or removed from the system.
+
+**Part numbers:**
+- Bottom panel: `6-39-X5803-013`
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 5 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>
+
+### Steps to remove the bottom cover:
+
+1. Place the machine lid-side down.
+    - Use a soft surface (such as a towel) to avoid scratches.
+2. Remove the 14 bottom panel screws.
+
+![Bottom panel screws](./img/bottom-panel-screws.webp)
+
+3. Pull the bottom panel off, starting from the back.
+
+## Replacing the battery:
+
+The battery provides primary power whenever the system is unplugged.
+
+**Part numbers:**
+- The battery's model number is `X580BAT-8-98`.
+- The battery's original part number is `6-87-X580S-94J01`.
+- Third-party battery sellers may list one or both of these numbers, and may offer other compatible part numbers with the same model number. You can also [contact System76](https://support.system76.com) to purchase a replacement battery.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 10 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to replace the battery:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Remove the four screws along the top of the battery.
+
+![Battery screws](./img/battery.webp)
+
+3. Unplug the white connector connecting the battery to the motherboard.
+4. Remove the battery, starting from the top edge with screw holes.
+5. When putting in the new battery, start with the plastic tabs along the bottom edge (opposite from the screw holes.)
+6. When plugging in the new battery, the red wire on the connector goes on the left, and the black wire goes on the right.
+
+## Replacing the RAM:
+
+The Bonobo WS 16 supports up to 192GB (4x48GB) of DDR5 SO-DIMMs running in dual-channel configuration at 5600MHz. The supported configurations are two or four SO-DIMMs (one or three SO-DIMMs are not supported configurations). If you've purchased new RAM, need to replace your RAM, or are reseating your RAM, follow these steps.
```

Comment:
I think single-SODIMM configs are supported, but only in certain slots. Would it be helpful to include a slot order for RAM installation? It's not so bad when there are only two slots, but four gets a little more complicated. Here's the one from the manual, for reference:
![Screenshot_2025-07-07_09-29-19](https://github.com/user-attachments/assets/f71b4cd2-40e0-41d2-80f2-b8075b3b9b43)

Also, I'm not sure the speed goes up to 5600MHz on this machine? If not, this also should be updated in README.md